02_bubble/nicholascross/maint_test.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestBubbleSort(t *testing.T) {
+	expectSorted([]int{1, 2, 3, 4, 5}, []int{1, 2, 3, 4, 5}, bubble, t)
```

Comment:
IMO it's nicer to have tests self contained and not calling other functions.
Table driven tests is the idiomatic go way of avoiding repetition here:
- [slides sample](https://github.com/anz-bank/go-slides/blob/master/samples/testing/basic/cases_test.go)
- [Dave Cheney post](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests)